<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="insertCard">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>241</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Syötä kortti</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_readRfid">
           <property name="text">
            <string>Lue kortti</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_card1">
           <property name="text">
            <string>Kortti 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_card2">
           <property name="text">
            <string>Kortti 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="login">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>60</y>
          <width>211</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SYÖTÄ PIN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>180</y>
          <width>239</width>
          <height>166</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_0">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_enter">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>370</y>
          <width>72</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>ENTER</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_clear">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>370</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>CLEAR</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pinLineEdit">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>140</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
       <widget class="QLabel" name="labelLoginError">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>440</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="card">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>130</y>
          <width>261</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SELECT CARD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="debitButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>DEBIT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="creditButton">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>240</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>CREDIT</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
